explainer.md

```diff
@@ -605,6 +605,29 @@ xrSession.addEventListener('resetpose', xrSessionEvent => {
 });
 ```
 
+### Page navigation
+
```

Comment:
This section doesn't address how the user agent behaves before/during the navigation other than to track "when a page navigates without ending an active exclusive session first."

Specifically, the UA needs to know whether to remain in XR presentation or transition to 2D. As written, I think the UA would always need to remain in XR presentation until "after all listeners for the `window`'s `load` event have fired." That would be different from navigating from all other pages. We might need some type of HTTP header to trigger this behavior. We should talk to navigation folks.